standalone/source/main.cpp

```diff
@@ -7,47 +7,47 @@
 #include <unordered_map>
 
 auto main(int argc, char** argv) -> int {
```

Comment:
main.cpp:9:6: warning: an exception may be thrown in function 'main' which should not throw exceptions [bugprone-exception-escape]
auto main(int argc, char** argv) -> int {
     ^

Reply:
leave it as demo only!